demos/image_retrieval_demo/python/image_retrieval_demo.py

```diff
@@ -134,7 +134,6 @@ def main():
 
     frames_processed = 0
     presenter = monitors.Presenter(args.utilization_monitors, 0)
-    video_writer = cv2.VideoWriter()
```

Comment:
Why move this? For `place_recognition_demo.py` it stays there.

Reply:
Updated.